notebooks/02-ingestion/05-working-with-nested-columns.ipynb

```diff
@@ -326,6 +326,44 @@
     "resp.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29 - Dynamic `path` in JSON_VALUE\n",
+    "JSON_VALUE and JSON_QUERY functions can now use a dynamic `path` parameter by specifying an expression instead of a literal.\n",
+    "\n",
```

Comment:
Add an example here before the commentary - I'm not clear what this means at this point.

Reply:
The example of JSON_VALUE using a literal for the "path" parameter is in the cell right above this. This is just a different, programmatic way of specifying which field in the JSON will be extracted.